docs/content/docs/data-structures/tree.md

```diff
@@ -547,7 +545,7 @@ on<K extends keyof TreeChangeEvents>(
 	): () => void;
 ```
 
-`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes. `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
+`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes (values of the fields in the node). `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
 
 An `event` object is automatically passed to the `listener`. It has one member:
```

Comment:
We should remove lines 550-552, `event.target` used to exist on a previous version of the events but does not exist on the current ones. We'll probably re-add it, but this document should not talk about it yet.

Reply:
@rohandubal let's address this before merging.